app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+    for someone in people:
+        name = someone["name"]
+        age = someone["age"]
+        person = Person(name, age)
+        person_list.append(person)
+
+    for someone in people:
```

Comment:
The second loop starting at this line is unnecessary. You can set the `wife` and `husband` attributes in the first loop itself. This will make the code more efficient and concise. According to the checklist, you should avoid unnecessary nested structures and loops.